spec/services/food_service_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe FoodsService, :vcr do
+  it "search by keyword" do
+    service = FoodsService.new
+    foods = service.search_by_keyword("sweet potatoes")[:foods]
+
+    expect(foods).to be_an(Array)
+    expect(foods.first).to be_a(Hash)
+    expect(foods.first).to have_key(:ingredients)
```

Comment:
It's a good idea to just check that all the necessary keys are there. Consider using `include` to test the presence of all the keys you need to use in the PORO